src/android/BLECentralPlugin.java

```diff
@@ -163,6 +163,17 @@ public class BLECentralPlugin extends CordovaPlugin {
     CallbackContext locationStateCallback;
     BroadcastReceiver locationStateReceiver;
 
+    public void setPairingCallback(PairingCallback pairingCallback) {
+        this.pairingCallback = pairingCallback;
+    }
+
+    PairingCallback pairingCallback;
+
+    interface PairingCallback{
```

Comment:
Introduced a callback for callingGattConnect from main flow after bonding is completed.